cmd/openfish/middleware/authentication.go

```diff
@@ -38,12 +38,24 @@ import (
 	"fmt"
 
 	"github.com/ausocean/openfish/cmd/openfish/api"
-	"github.com/ausocean/openfish/cmd/openfish/entities"
 	"github.com/ausocean/openfish/cmd/openfish/services"
+	"github.com/ausocean/openfish/cmd/openfish/types/role"
 	"github.com/gofiber/fiber/v2"
 	"google.golang.org/api/idtoken"
```

Comment:
Order imports, std, 3rd party, ausocean, separated by a line.

Reply:
When I run go fmt ./cmd/openfish it reorders them into this order, so I would prefer not to change that.